provider/oci/images.go

```diff
@@ -140,15 +152,16 @@ func (t byVersion) Less(i, j int) bool {
 	return true
 }
 
+type imageMap map[corebase.Base]map[string][]InstanceImage
```

Comment:
Given this is a complex type, please comment on what each value holds, "string" could be anything.

Reply:
you're absolutely right. Let me know if the description I added looks good or you would rephrase it